go.mod

```diff
@@ -4,15 +4,15 @@ go 1.23.0
 
 require (
 	github.com/blang/semver/v4 v4.0.0
-	github.com/ceph/go-ceph v0.30.0
```

Comment:
why are go.mod and go.sum changed too? minimally some packages were updated which aren't related with changes in PR.

Reply:
Updated description to reflect the changes